src/main/java/core/basesyntax/service/ReportWriter.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportWriter {
```

Comment:
create interface FileWriter for example
```suggestion
public class ReportWriter implement FileWriter {
```